### PR TITLE
Adds `longform` option to `prettytime`

### PR DESCRIPTION
**Pull request by glwagner**
Useful if we want to print a lot of time values in an array.

### CODE REVIEW

File: test/test_utils.jl [hunk 1]
```diff
@@ -1,3 +1,5 @@
+include("dependencies_for_runtests.jl")
+
```
Comment (navidcy) [inline test/test_utils.jl:2]: `runtests.jl` includes that and then `test_utils.jl` re-includes it?
what about function re-definitions? I think in Julia 1.7 we may be getting warnings for that.

**Review by navidcy — APPROVED** [review]
Can you put in the first comment an example of what an output with `longform` looks like? Is it just `hrs -> hours`, etc?

File: test/test_utils.jl (hunk 1 above)
Comment (glwagner) [reply test/test_utils.jl:2]: Dang

Comment (glwagner) [reply test/test_utils.jl:2]: The purpose here is to include the test file at the REPL so we can debug a test with `Revise`. Maybe there's a better way. Or, we can remove all of these, and put them in just when we need to do that debugging.

**Comment by glwagner** [comment]
> Can you put in the first comment an example of what an output with `longform` looks like? Is it just `hrs -> hours`, etc?

Yaa. Longform is default, retains current behavior. Shortform could be used for showing an array of times, eg

```julia
julia> t = [0, 10minutes, 3days, 5years]
4-element Vector{Float64}:
      0.0
    600.0
 259200.0
      1.5768e8

julia> prettytime.(t)
4-element Vector{String}:
 "0 seconds"
 "10 minutes"
 "3 days"
 "5 years"

julia> prettytime.(t, false)
4-element Vector{String}:
 "0 s"
 "10 m"
 "3 d"
 "5 yrs"
```

**Comment by glwagner** [comment]
We can also use the tools in `Unitful.jl` for this kind of thing, so maybe we'll transition to that if we resolve #1462 .

**Pull request merged**